### PR TITLE
Add slash command handling

### PR DESCRIPTION
**Pull request by tomkettle1**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -5,14 +5,57 @@ import dotenv from 'dotenv';
...
+client.login(process.env.DISCORD_TOKEN);
+
+async function registerSlashCommands(message: Discord.Message){
+    // see https://deploy-preview-638--discordjs-guide.netlify.app/interactions/registering-slash-commands.html
+    
+    // IMPORTANT: Remember to give the bot perms to create slash commands w/ the oauth page (application.commands privileges)
+
+    await client.application?.fetch();
+    
+    // Please add your command name & desc. here. CAREFUL! As of right now it looks like a huge pain to delete these, so add carefully
```
Comment (dislocated-su) [inline src/index.ts]: Doesn't `commands.set()` override the list of commands? If so this shouldn't be an issue.

File: src/index.ts [hunk 2]
```diff
@@ -5,14 +5,57 @@ import dotenv from 'dotenv';
 
 dotenv.config()
 
+const DevIDs = ["170960451704717312"/*dislocated*/, "428934780764160010"/*tomkettle*/]
```
Comment (dislocated-su) [inline src/index.ts:8]: Might be worth moving this to .env

**Review by dislocated-su — APPROVED** [review]
Good. There are potential improvements that can be made later on

File: src/index.ts (hunk 1 above)
Comment (tomkettle1) [reply src/index.ts]: ah, its replacing the whole array? That'd make more sense than combining concidering its 'set'

Comment (tomkettle1) [reply src/index.ts]:
```suggestion
    // Please add your command name & description here. 
```

File: src/index.ts (hunk 2 above)
Comment (tomkettle1) [reply src/index.ts:8]: We can, yh, that might be tidier. Is there a nice way to hold arrays in .env or do you have to jank it?

**Pull request merged**